rust/worker/src/execution/operators/brute_force_knn.rs

```diff
@@ -169,9 +166,7 @@ impl Operator<BruteForceKnnOperatorInput, BruteForceKnnOperatorOutput> for Brute
             // Empty allowed list is passed when where filtering is absent.
             // TODO: This should not need to use merged_user_id, which clones the id.
             if !input.allowed_ids.is_empty()
-                && !input
-                    .allowed_ids_brute_force
-                    .contains(&log_record.merged_user_id())
+                && !input.allowed_ids.contains(&log_record.merged_user_id())
```

Comment:
i remember you added this, how come we can remove it? just want to make sure I understand this

Reply:
Let's talk offline on this